dotnet/samples/GettingStartedWithProcesses/Events/CommonEvents.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+namespace Events;
+
+/// <summary>
+/// Processes Events used in multiple Processes samples.<br/>
+/// </summary>
+public static class CommonEvents
```

Comment:
hopefully with the steps reorg directory structure this will help, also leaving some common steps that would be the only ones making use of the CommonEvents. Perhaps with the upcoming samples we can explore how to deal with events.

Idea for the future: since it seems like events are linked to a step, so maybe they should be inside each step class def?
So then out of the box, you could potentially inspect what are the possible all the event outputs of a step without looking into the code